docs/reference/snapshot-restore/index.asciidoc

```diff
@@ -0,0 +1,87 @@
+[[snapshot-restore]]
+= Snapshot and restore
+
+[partintro]
+--
+
+// tag::snapshot-intro[]
+A snapshot is a backup taken from a running Elasticsearch cluster. You can take
+a snapshot of individual indices or of the entire cluster and store it in a
+repository on a shared filesystem, and there are plugins that support remote
+repositories on S3, HDFS, Azure, Google Cloud Storage and more.
```

Comment:
This sentence is a bit long. I would mention the plugins in a separate sentence.

Reply:
Reworked.